frontend/src/fixtures/earthquakesFixtures.js

```diff
@@ -0,0 +1,35 @@
+const earthquakesFixtures = {
+    oneEarthquake: {
+        "id": 1,
+        "title": "M 2.2 - 10km ESE of Ojai, CA",
+        "mag": 2.16,
+        "place": "10km ESE of Ojai, CA",
+        "time": 1644571919000
```

Comment:
In the backend, I made the decision to make the `time` field a `String`, because I didn't want a `Long` to overflow is we're storing an epoch. This isn't really important, because the backend will serialize everything anyway. Just wanted to point it out.